tests/AutoloadTest.php

```diff
@@ -28,7 +28,7 @@
 
 namespace EPF;
 
-require_once("Autoload.php");
+require_once("EPF/Autoload.php");
```

Comment:
We should use __NAMESPACE__ here. So that the test is more specificity agnostic.